src/components/Cars/CarCard.tsx

```diff
@@ -1,66 +1,119 @@
-import {Link} from "react-router-dom";
-import React, {FC} from "react";
-import {CarStatus, ICar} from "../../types/types";
-import {useSelector} from "react-redux";
-import {selectLoggedIn} from "../../slices/UserSlice";
+import { FC } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { selectLoggedIn } from '../../slices/UserSlice';
+import { CarStatus, ICar } from '../../types/types';
 
 export interface CarCardProps {
-    car: ICar | null,
-    startRent?: () => void,
-    startBook?: () => void,
-    finishRide?: () => void,
-    pauseRent?: () => void,
+  car: ICar | null;
+  startRent: () => void;
+  startBook: () => void;
+  finishRide: () => void;
+  pauseRent: () => void;
 }
 
 const CarCard: FC<CarCardProps> = ({
-                                       car,
-                                       startRent,
-                                       startBook,
-                                       finishRide,
-                                       pauseRent,
-                                   }) => {
-    const loggedIn = useSelector(selectLoggedIn);
+  car,
+  startRent,
+  startBook,
+  finishRide,
+  pauseRent,
+}) => {
+  const loggedIn = useSelector(selectLoggedIn);
 
-    return (
-        <div className="card h-100">
-            <Link to={"/cars/" + car?.id}><img className="card-img-top" src={car?.imgUrl} alt=""/></Link>
-            <div className="card-body text-center d-flex flex-column">
-                <h6 className="card-title">{car?.mark} {car?.model}</h6>
-                <div>
-                    <p>Rent <span style={{fontWeight: "bolder"}}>{car?.rentPricePerHour}</span> $/h</p>
-                </div>
-                <div>
-                    <p>Book <span style={{fontWeight: "bolder"}}>{car?.bookPricePerHour}</span> $/h</p>
-                </div>
+  return (
+    <div className="card h-100">
+      <Link to={`/cars/${car?.id}`}>
+        <img alt="" className="card-img-top" src={car?.imgUrl} />
+      </Link>
+      <div className="card-body text-center d-flex flex-column">
+        <h6 className="card-title">
+          {car?.mark} {car?.model}
+        </h6>
+        <div>
+          <p>
+            Rent{' '}
+            <span style={{ fontWeight: 'bolder' }}>
+              {car?.rentPricePerHour}
+            </span>{' '}
+            $/h
+          </p>
+        </div>
+        <div>
+          <p>
+            Book{' '}
+            <span style={{ fontWeight: 'bolder' }}>
+              {car?.bookPricePerHour}
+            </span>{' '}
+            $/h
+          </p>
```

Comment:
Code repeat, it is better to create new component and pass currency as one of the props.